documents/operating.md

```diff
@@ -0,0 +1,329 @@
+Although previously thought impossible, an ancient ritual found buried
+in the ruins of the Lawyer Cave describes the process of creating new
+BIFFUD Corporate Overlords.
+
+Any Corporate Overlord may nominate the addition of a new Overlord during
+a plotting session.  If the nomination is seconded, the ritual begins.
+
+1. The Corporate Overlords are presented with a biography of the candidate.
+2. The candidate submits a piece of original poetry or other creative work
+for the Overlords' enjoyment and consideration.
+3. A vote is held.
+
+If 70% or more of the Corporate Overlords approve the addition, then after the
+plotting session ends the new Corporate Overlord is onboarded and ownership of
+the company is redistributed so that each overlord owns an equal share.
+
+
+### Removing Corporate Overlords
+A Corporate Overlord can recuse itself from BIFFUD at any time by dying, uploading
+its consciousness to the cloud, or submitting a letter of resignation to both
+the President of BIFFUD and the President of the United States.
+
+Corporate Overlords can also vote to remove a peer from the organization if
+any of the following conditions apply:
+
+- The Corporate Overlord fails to attend 50% of the monthly plotting sessions
+in a given year without a note from a family member explaining its absence.
+- The Corporate Overlord has violated a serious federal law in the past year.
+- The Corporate Overlord is hated or despised.
+
+The removal of a corporate overlord can be nominated during a plotting session
+and requires 70% approval or more from the other Corporate Overlords.
+
+Overlords are entitled to a pro-rated profit share for their tenure in a given
+year, as well as ongoing profit-share according to their ownership of various
+BIFFUD projects.
+
+---------------------------------------------------------------------
+SECTION 3: 🤔🤔🤔 Projects 🤔🤔🤔
+---------------------------------------------------------------------
+- [x] Types of project
+- [x] Evaluating ideas
+- [x] IP (Intellectual Property / Internet Protocol)
+- [x] Open source
+
+### Types of Project
+A. **Grant Funded:** projects funded by grants from third parties are executed
+by the BIFFUD member who has led the relationship with the funders, or by an
+alternative BIFFUD member as delegated by that applicant.  Money associated with
+a grant is to be distributed as per the grant's agreement.
+
+B. **Member Owned:** projects that are led or adopted by BIFFUD members.  The
+owners of a project must all be BIFFUD members, and the percentages and specific
+terms of ownership should be define by those members as part of their application
+for the adoption of the idea under BIFFUD.
+
+C. **BIFFUD Core:** projects that are taken on by the entire corporation.  These
+might be projects utilized by the Corporate Overlords to execute on company policy,
+or simply be projects that the Corporate Overlords decide to develop without a concept
+of individual ownership.
+
+
+### Evaluating Ideas
```

Comment:
@knowtheory @crupar This sounds like a a good example of tension between wanting both the chaos and the collective components of BIFFUD!

Just to explain the thinking behind the proposed structure:

1) It sounded kinda fun to force us to talk a little bit as a group about every single potential new project. I think that would also add value to the projects through the feedback process.

2) Having one person responsible for that decision seems to lose some accountability / gives total control to brand identity to that person since they decide what the projects are.  I'm a little less excited about that just because I'm worried we lose some of the "collective" angle of the group.

3) There are some decisions to be potentially made when approving a project (though I suppose that's only if people want special treatment or a budget)

It's worth noting that for some of the more menial things (e.g. some kinds of expenses) there is a proposal of some deputizing to the treasurer.

Is the concern the fact that we wouldn't be able to start a project without a board vote, or is there other concern?  I was figuring people would just start making code in their own githubs and then raise the project at the next CO meeting.